cms/templates/settings_graders.html

```diff
@@ -58,7 +58,7 @@ <h2 class="title-2">${_("Overall Grade Range")}</h2>
           <ol class="list-input">
             <li class="field" id="field-course-grading-range">
               <div class="grade-controls course-grading-range well">
-                <a href="#" class="new-grade-button"><span class="icon fa fa-plus"></span></a>
+                <a href="#" class="new-grade-button"><span class="icon fa fa-plus" aria-hidden="true"></span></a>
```

Comment:
I think the link might need a title or hover-over text as well. It was pretty confusing to me what the button did until I clicked it and saw the grade range change.